x-pack/filebeat/input/httpjson/input_test.go

```diff
@@ -1191,6 +1235,13 @@ func TestInput(t *testing.T) {
 					}
 				}
 			}
+			if tc.baseConfig["verifyfilepath"] != nil {
+				if _, err := os.Stat("../../logs/httpjson/http-request-trace-httpjson-foo-eb837d4c-5ced-45ed-b05c-de658135e248_https_somesource_someapi.ndjson"); err == nil {
```

Comment:
This seems a little disjoint to have the path hard-coded into the test case runner. Perhaps the path could be specified as part of the test case config and if non-empty then perform the check?